### DISCUSSION TITLE
Add a list of current subscriptions.

### DISCUSSION THREAD

**Discussion by redwizard42** [RetroAchievements/RAWeb#1414] [Ideas]
I am subscribed to a wide variety of tickets, forum comments, wall comments etc... It'd be nice to see a list of what all I am subscribed to, with the ability to unsubscribe on that list. I don't want to unsubscribe from my own stuff usually, so this would be much preferred over simply an 'unsubscribe all'.  Especially useful because you get auto-subscribed for commenting on things the first time and often forget until you get an email from this much later on.

It might also be nice to see that I am subscribed to something, but have email notifications turned off for that type of thing (sort of like a muted subscription, I guess).

Other possibilities on this list: "unsubscribe to all forum comments" and other similar things.  This should definitely have a "Are you sure?" prompt.

**Comment by luchaos** [comment]
Great idea!

A first attempt for refactoring subscriptions was made here https://github.com/RetroAchievements/RAWeb/pull/1025

It was planned to add said overview list for all subscriptions to make it easier to manage. Due to how subscriptions are implemented it would require some additional work to persist the existing "derived" subscriptions first to make it easier to manage. Some subscriptions are implicit and hard to reason about.
